test/parallel/test-util-inspect.js

```diff
@@ -235,3 +235,17 @@ if (typeof Symbol !== 'undefined') {
   assert.equal(util.inspect(subject, options), '[ 1, 2, 3, [length]: 3, [Symbol(symbol)]: 42 ]');
 
 }
+
+// test Set
+assert.equal(util.inspect(new Set), 'Set {}')
```

Comment:
Yeah, for some reason that choice was made a long time ago. Wouldn't call it "normal" though :)